ftw/htmlblock/upgrades/20170428172400_installiframefix/upgrade.py

```diff
@@ -0,0 +1,9 @@
+from ftw.upgrade import UpgradeStep
+
+
+class Installiframefix(UpgradeStep):
+    """Installiframefix.
```

Comment:
Please change to a human readable sentence (will be presented to the user in the Plone control panel).